lib/solid_errors/db/migrate/20240716154238_add_prev_resolved_at_to_solid_errors.rb

```diff
@@ -0,0 +1,5 @@
+class AddPrevResolvedAtToSolidErrors < ActiveRecord::Migration[6.1]
+  def change
+    add_column :solid_errors, :prev_resolved_at, :timestamp
```

Comment:
Is it `timestamp` and not `datetime`? `timestamp` is not one of the basic types used by `add_column`:

> The `type` parameter is normally one of the migrations native types, which is one of the following: `:primary_key`, `:string`, `:text`, `:integer`, `:bigint`, `:float`, `:decimal`, `:numeric`, `:datetime`, `:time`, `:date`, `:binary`, `:boolean`.
— from: https://api.rubyonrails.org/v6.1.1/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_column

Reply:
`datetime` works just fine. I was thinking of it along the lines of the created_at and updated_at `timestamp` fields.